src/Orleans/Streams/Predicates/IStreamNamespacePredicate.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Orleans.Streams
+{
+    public interface IStreamNamespacePredicate
+    {
+        bool IsMatch(string streamNamespace);
```

Comment:
nit on naming : bool Match(string namespace). IsMatch is weird to me grammatically.